web/src/css/flowtable.less

```diff
@@ -114,16 +114,19 @@
         }
     }
     .col-method {
-        width: 60px;
+        width: auto;
```

Comment:
What's the motivation for adjusting the width here? IMO this looks less desirable on certain screen sizes now.